### PR TITLE
FINAL: SC2-4-2-title

### PR DESCRIPTION
**Pull request by jkodu**
fixes: 
- editorial comments.
- added test cases & assets.
- fix broken links.
- tackled comments in issue - https://github.com/auto-wcag/auto-wcag/issues/99

### CODE REVIEW

**Comment by WilcoFiers** [comment]
+jen as reviewer

File: _drafts/SC2-4-2-title.md [hunk 1]
```diff
@@ -50,3 +51,60 @@ _There are no major accessibility support issues known for this rule._
...
+</html>
+```
+
+```html
+<html>
+  <title>This page gives a title to an iframe</title>
+  <iframe src="../test-assets/sc2-4-2-title-page-without-title.html"></iframe>
+</html>
+```
+
```
[the remark this answers is not in the document]
Comment (jkodu) [reply _drafts/SC2-4-2-title.md]: Added test case. Please review again.

**Comment by jkodu** [comment]
@ShadowBB @kasperisager @WilcoFiers @cpandhi @JenniferChadwick @carlosapaduarte 
Please review again.

**Comment by WilcoFiers** [comment]
Should the rule ID be page-has-title?

**Comment by jkodu** [comment]
Have made necessary amends. Please review again.

File: _rules/SC2-4-2-page-has-title.md [hunk 2]
```diff
@@ -88,6 +88,17 @@ _There are no major accessibility support issues known for this rule._
...
+<html>
+  <head>
+    <title> <!-- this page has an empty first title --> </title>
+  </head>
+  <body>
+    <title>Title of the page.</title>
+  </body>
+</html>
+```
+
```
Comment (kasperisager) [inline _rules/SC2-4-2-page-has-title.md]: This test will currently fail the rule, not pass:

> The first title element contains non-empty text.

Comment (jkodu) [reply _rules/SC2-4-2-page-has-title.md]: @kasperisager - Good spot. Amended.

**Pull request merged**